src/plugins/plot/src/plot/MCTTicksController.js

```diff
@@ -166,6 +174,11 @@ define([
         return ticks(range.min, range.max, number);
     };
 
+    MCTTicksController.prototype.updateTicksForceRegeneration = function () {
+        this.forceRegeneration = true;
+        this.updateTicks();
```

Comment:
You could pass a `forceUpdate` flag to `updateTicks` from here, and bypass `shouldRegenerateTicks` altogether, rather than setting an instance variable.

Reply:
done